lib/ansiblelint/cli.py

```diff
@@ -161,42 +161,37 @@ def get_cli_parser() -> argparse.ArgumentParser:
 
 
 def merge_config(file_config, cli_config) -> NamedTuple:
+    bools = (
+        'display_relative_path',
+        'parseable',
+        'quiet',
+        'use_default_rules',
+    )
```

Comment:
Any reason to not include 'parseable_severity' here?

Reply:
Only lack of attention. Thanks fixed.